tests/test_routes.py

```diff
@@ -67,8 +67,16 @@ def test_get_inventory_list(self):
 
     def test_get_inventory(self):
         """It should Get a single Inventory item"""
-        self.client.get(f"{BASE_URL}/1")
-        # do some asserts
+
+        test_inventory = InventoryFactory()
+        test_inventory.create()
+        response = self.client.get(f"{BASE_URL}/pid/{test_inventory.pid}/condition/{test_inventory.condition.value}")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        data = response.get_json()
+        self.assertEqual(data["name"], test_inventory.name)
+
+
+
```

Comment:
Add test case for invalid condition id and status 404 not found for higher coverage, use the same code as 
test_update_inventory_no_item_present
test_update_inventory_bad_condition_id